src/main/minio.js

```diff
@@ -572,7 +570,7 @@ export class Client {
   // * `region` _string_ - region valid values are _us-west-1_, _us-west-2_,  _eu-west-1_, _eu-central-1_, _ap-southeast-1_, _ap-northeast-1_, _ap-southeast-2_, _sa-east-1_.
   // * `makeOpts` _object_ - Options to create a bucket. e.g {ObjectLocking:true} (Optional)
   // * `callback(err)` _function_ - callback function with `err` as the error argument. `err` is null if the bucket is successfully created.
-  makeBucket(bucketName, region="", makeOpts={}, cb) {
+  makeBucket(bucketName, region, makeOpts={}, cb) {
```

Comment:
You are making the optional `region` parameter mandatory. This would break the existing scripts out there, and it would cause regression problems.

Reply:
Thank you. 
I have ran the test locally and it completed successfully :
```
 219 passing (39s)
  3 pending

[16:45:12] Finished '<anonymous>' after 39 s
[16:45:12] Finished 'functionalTest' after 41 s

```

The default region is specified as in mino-go
```
// If location is empty, treat is a default region 'us-east-1'.
	if location == "" {
		location = "us-east-1"
		// For custom region clients, default
		// to custom region instead not 'us-east-1'.
		if c.region != "" {
			location = c.region
		}
	}
```


I ran further js tests to pass 
- `""`
- `us-east-1`
-  or use a different region from server region which should retry and create bucket as configured in server
( so the client can pass any one of these)

Please guide me if i have missed any scenario. and thank you so much for the review.